### DISCUSSION TITLE
Deployment to existing RG and AAD

### DISCUSSION THREAD

**Discussion by ymac3** [microsoft/PubSec-Info-Assistant#742] [Q&A] [answered]
Does anyone know how to deploy to an existing resource group and using the existing AAD created by the AD owner, the deployment account has no access to Azure EntraID.

**Comment by dayland** [comment, marked as answer]
https://github.com/microsoft/PubSec-Info-Assistant/discussions/457 and https://github.com/microsoft/PubSec-Info-Assistant/discussions/721

**Reply by ymac3** [reply]
thanks mate!
